after.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# If you would like to do some extra provisioning you may
+# add any commands you wish to this file and they will
+# be run after the Homestead machine is provisioned.
```

Comment:
We certainly can. If we need it in the future we can just re-run the Homestead `make` command or just simply drop this file into the root and Homestead will pick it up and execute it automatically. Will remove it for now.